client/src/components/_common/UserSearchbar/UserSearchbar.js

```diff
@@ -14,42 +14,42 @@ const UserSearchbar = ({
   searchResults,
```

Comment:
looks like we need to remove this file from the main 737 branch, but could you remove it here in the meantime?